resources/RESOURCES.md

```diff
@@ -0,0 +1,93 @@
+• [Sqlite](https://www.npmjs.com/package/sqlite) (single file database)
+• [keyv](https://www.npmjs.com/package/keyv) (key-value storage)
+• [Sequelize](https://www.npmjs.com/package/sequelize) (object relational mapper)
+• [Mikro-ORM](https://www.npmjs.com/package/mikro-orm) (object relational mapper - type support)
+• [TypeORM](https://www.npmjs.com/package/typeorm) (object relational mapper - type support)
+• [postgres](https://www.npmjs.com/package/postgres) (postgres client for node)
+• [PM2](https://www.npmjs.com/package/pm2) (process manager)
+• [chalk](https://www.npmjs.com/package/chalk) (colored logging)
+• [winston](https://www.npmjs.com/package/winston) (logging provider)
+• [node-cron](https://www.npmjs.com/package/node-cron) (task scheduling)
+• [node-schedule](https://www.npmjs.com/package/node-schedule) (task scheduling)
+• [cheerio](https://www.npmjs.com/package/cheerio) (html parsing)
+_ _
+
+**Debugging:**
+For when you can't track that one pesky error down:
+• [Node.js](https://nodejs.org/en/docs/guides/debugging-getting-started/)
+• [Visual Studio Code](https://code.visualstudio.com/docs/nodejs/nodejs-debugging)
+• [Linters](https://discordjs.guide/preparations/setting-up-a-linter.html) (syntax errors and code consistency)
+• [MDN list of common errors](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors)
+• [Guide on common errors and how to deal with them](https://discordjs.guide/popular-topics/errors)
+• Discord.js: attach the following two listeners to your Client instance (outside of any other listener scopes)
+```js
+client.on('debug', console.log)
+      .on('warn', console.log)
+```
+*Note: if you initialize your Client as `bot`or other identifiers you need to use these instead of `client`.*
+_ _
+**Affordable Hosting providers:**
+• [OVH](https://www.ovh.com/us/vps/) Starting at $3.35/mo for 1 core, 2GB RAM, 20GB SSD
+• [DigitalOcean](https://www.digitalocean.com/) Starting at $5/mo for 1 core, 1GB RAM, 25GB SSD
+• [Linode](https://www.linode.com/) Starting at $5/mo for 1 core, 1GB RAM, 25GB SSD
+• [Vultr](https://www.vultr.com/) Starting at $2.50/mo for 1 core, 512MB RAM, 10GB SSD
+• [AWS Lightsail](https://amazonlightsail.com/) Starting at $3.50/mo (first month free) for 1 core, 512MB RAM, 20GB SSD
+• [Time4VPS](https://www.time4vps.eu/) Starting at €3.99/month for 1 core, 2GB RAM, 20GB SSD
+• [VIRMACH](https://virmach.com/) Full Windows and Linux Desktop VPS starting at $7/mo and $1/mo respectively
+• [Netcup](https://www.netcup.eu/) Starting at €2.62/mo for 1 core, 2GB RAM, 20GB SSD
+• [GitHub education](https://education.github.com/pack) Free credit and other offers for students
+• [Microsoft students](https://azure.microsoft.com/en-us/free/students/) Free credit for students
+• [Google Cloud](https://cloud.google.com/free) free tier
+_ _
+
+⬆ [jump to the top](https://canary.discordapp.com/channels/222078108977594368/729580210634358804/753543768250122312)
```

Comment:
We can't re-use the same link, let's use the `0` shortcut (and hope they fixed the jump bug 😓)

```suggestion
⬆ [jump to the top](https://canary.discordapp.com/channels/222078108977594368/729580210634358804/0)
```